src/pages/JobCalendar.page

```diff
@@ -134,35 +134,35 @@
+                                var intTotalVolunteers = result[i].{!JSENCODE(ns)}Total_Volunteers__c;
                                 if (intTotalVolunteers == null) intTotalVolunteers = 0;
-                                var intStillNeeded = result[i].{!ns}Number_of_Volunteers_Still_Needed__c;
+                                var intStillNeeded = result[i].{!JSENCODE(ns)}Number_of_Volunteers_Still_Needed__c;
                                 if (intStillNeeded == null) intStillNeeded = 0;
                                 var strShiftNumbers = '{!$Label.labelShiftNumbers}';
                                 strShiftNumbers = strShiftNumbers.replace('{0}', intTotalVolunteers);
                                 strShiftNumbers = strShiftNumbers.replace('{1}', intStillNeeded);
                                 
-                                var strCampaignId = new String(result[i].{!ns}Volunteer_Job__r.Campaign__c);
+                                var strCampaignId = new String(result[i].{!JSENCODE(ns)}Volunteer_Job__r.Campaign__c);
                                 var strJobId = new String(jobId);
                                 var strShiftId = new String(result[i].Id);                              
                                 var strUrl;
                                 
                                 if ({!fWeb}) {
-                                    strUrl = '{!$Site.CurrentSiteUrl + IF(fPersonalSite, ns+'PersonalSiteJobListing', ns+'VolunteersJobListingFS')}' +
+                                    strUrl = '{!JSENCODE($Site.CurrentSiteUrl + IF(fPersonalSite, ns+'PersonalSiteJobListing', ns+'VolunteersJobListingFS'))}' +
                                         '?Calendar=1&volunteerShiftId=' + strShiftId + '&jobId=' + strJobId +
                                         '&dtMonthFilter=' + strApexDateUTC(dtStart) +
-                                        '{!IF(strParams != null, '&' + strParams, '')}';
+                                        '{!JSENCODE(IF(strParams != null, '&' + strParams, ''))}';
                                 } else if ({!NOT(fPrint)}) {
                                     strUrl = '/' + strShiftId;
                                 }                                
 
                                 // note that we lop off the timezone, to avoid FullCalendar from translating it to the current browser locale.                                                
                                 shifts.push({
-                                    title: result[i].{!ns}Volunteer_Job__r.Name + (intStillNeeded == 0 ? ' ' + 
+                                    title: result[i].{!JSENCODE(ns)}Volunteer_Job__r.Name + (intStillNeeded == 0 ? ' ' + 
                                         '{!$Label.labelCalendarShiftFull}' : ''),
-                                    description: result[i].{!ns}Description__c,
+                                    description: result[i].{!JSENCODE(ns)}Description__c,
```

Comment:
put this JSENCODE around the description which might contain quotes and slashes.